docs/schemas/itemRaw.json

```diff
@@ -0,0 +1,733 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "tas-answer",
+  "description": "An item anwser",
+  "definitions": {
+
+    "tas-file": {
+      "description": "Uploaded file",
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "string",
+          "media": {
+              "binaryEncoding": "base64"
+          }
+        },
+        "mimeType":   { "type": "string" },
+        "name":       { "type": "string" },
+        "via":        { "type": "string" },
+        "url":        { "type": "string" },
+        "confirmUrl": { "type": "string" }
+      }
+    },
+
+    "tas-stringAnswer": {
+      "description": "Schema for string answer",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Answer for this item",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-chronoAnswer": {
+      "description": "Schema for string answer",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Answer for this item",
+          "type": "string"
+        }
+      }
+    },
+
+    "tas-singleLineAnswer": {
+      "description": "Schema for singleLine answer",
+      "allOf": [
+         { "$ref": "#/definitions/tas-stringAnswer" },
+         { "properties": {
+             "type": {
+               "type": "string",
+               "enum": ["singleLine"]
+             }
+           }
+         }
+      ]
+    },
+
+    "tas-multilineAnswer": {
+      "description": "Schema for multiline answer",
+      "allOf": [
+         { "$ref": "#/definitions/tas-stringAnswer" },
+         { "properties": {
+             "type": {
+               "type": "string",
+               "enum": ["multiline"]
+             }
+           }
+         }
+      ]
+    },
+
+    "tas-numberAnswer": {
+      "description": "Schema for number answer",
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": ["number"]
+        },
+        "value": {
+          "description": "Answer for this item",
+          "type": "number",
+          "multipleOf" : 1,
+          "minimum": -2147483648,
```

Comment:
These constraints on the value ensure that the data will fit into a 32-bit integer.